src/Cd.java

```diff
@@ -0,0 +1,13 @@
+/**
+ * Created by sparatan117 on 11/28/16.
+ */
+//for a cd holds the artist name, album name, name of songs, how many songs, and the length of each song
+public class Cd {
+    String nameOfSong;
+    String artist;
+    String albumName;
+    int numberOfSongs;
+    int lengthOfSongInSeconds;
```

Comment:
Good that you are thinking about things like what the units are.  I would probably put songLength then put a comment right after explaining what the units are.